src/components/Stories/components/StoriesLayout/StoriesLayout.tsx

```diff
@@ -60,6 +60,11 @@ export const StoriesLayout = (props: StoriesLayoutProps) => {
                                             {currentStory.description}
                                         </div>
                                     )}
+                                    {currentStory.content && (
```

Comment:
Is it better to display only content, and if it is not defined, fallback to description?

Reply:
fixed